openshift-with-appstudio-test/e2e/periodic_test.go

```diff
@@ -170,6 +169,7 @@ func TestServiceRegistry(t *testing.T) {
 		state := sync.Map{}
 		var changed uint32
 		exitForLoop := false
+		timeoutChannel := time.After(90*time.Minute)
```

Comment:
@stuartwdouglas - I did bump the timeout down from 2 hours to 90 minutes (where the overall test is aborted by the framework after 3 hours).

I did this in part to have the test fail a bit sooner while the artifact cache hang is being debugged, but to let it run long enough to make sure the hang manifests.

Once the hang has been fixed, I certainly could see changing this back to 2 hours (I think we need a little buffer for cluster bringup and the initial build) if we get back to a point where things are fine but we determine we just need more time.